docs/contributing/reporting-bugs.md

```diff
@@ -41,13 +41,13 @@ Filing a GitHub issue works best when:
   This is important for developers to be able to fix the bug, and test that
   their fix worked.
 
-If all of the above accurately describe your situation, please file an issue!
+If all of the above accurately describes your situation, please file an issue!
```

Comment:
I read this as 'If all of the (bullet points) above accurately describe ..." and therefore would use the plural verb form. We could alternatively say, "If everything above accurately describes ..."

Reply:
Whoops, yup. I overthought and hypercorrected on that "all."